### PR TITLE
sg: let enterprise prefix be implicit, oss prefix explicit, add name truncation

### PR DESCRIPTION
**Pull request by jhchabran**
image

Quick fix that simply aliases some commands, at least it won't mess up users with small terminals. 

## Test plan

Tested locally

### CODE REVIEW

File: dev/sg/internal/run/logger.go [hunk 1]
```diff
@@ -25,10 +27,20 @@ func newCmdLogger(ctx context.Context, name string, out *output.Output) *process
...
-		out.Writef("%s%s[%+30s]%s %s", output.StyleBold, color, name, output.StyleReset, data)
+		// Some rare commands will have names larger than 15 chars, but that's fine.
+		// How to quickly check commands names: cue eval --out=json sg.config.yaml | jq '.commands | keys'
+		out.Writef("%s%s[%+15s]%s %s", output.StyleBold, color, name, output.StyleReset, data)
 	}
 
 	return process.NewLogger(ctx, sink)
 }
+
+func compactName(name string) string {
```
Comment (bobheadxi) [inline dev/sg/internal/run/logger.go:38]: Should we just change the names in sg.config.yaml instead? The discrepancy might be confusing

Comment (jhchabran) [reply dev/sg/internal/run/logger.go:38]: Yes, but if we do that, the service names really start to look confusing and unclear. This PR just buys us some time while we can think about how to fix this.

Comment (jhchabran) [reply dev/sg/internal/run/logger.go:38]: @bobheadxi context: https://sourcegraph.slack.com/archives/C01N83PS4TU/p1655729108306929 :)

**Comment by bobheadxi** [comment]
I think @efritz 's suggestion here is the way to go:

> also, I think our services don’t need the enterprise- prefix
>
> since we all use enterprise all the time, why not flip it around and name the oss services oss- and enterprise just $NAME

`sg start` already defaults to enterprise anyway, so the whole thing should be inverted

**Comment by jhchabran** [comment]
> I think @efritz 's suggestion here is the way to go:
> 
> 
> 
> > also, I think our services don’t need the enterprise- prefix
> 
> >
> 
> > since we all use enterprise all the time, why not flip it around and name the oss services oss- and enterprise just $NAME
> 
> 
> 
> `sg start` already defaults to enterprise anyway, so the whole thing should be inverted

I agree, I just didn't want to break the terminal for too many users before taking off 😊

**Comment by bobheadxi** [comment]
I'm working on the rename!

**Comment by efritz** [comment]
FTR, @eseliger made that suggestion!

**Comment by bobheadxi** [comment]
image

Tested `sg start` and `sg start oss`, and checked all the commandsets by hand

**Comment by bobheadxi** [comment]
Truncated example:

image

**Review by bobheadxi — APPROVED** [review]
giving myself the thumbs up

**Pull request merged**